src/js/components/ToggleGroup/stories/Icons.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Box, ToggleGroup } from 'grommet';
+import { List, MapLocation, Table } from 'grommet-icons';
+
+const optionsIcons = [
```

Comment:
I think this story should be the "simple." Then this file Icons story can go away.

Reply:
Done.